src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -97,27 +99,28 @@
                            (string/join "\n" config-files)))
         config-files))))
 
-(schema/defn ^:private config-from-cwd :- (schema/maybe BootstrapFiles)
+(schema/defn ^:private config-from-cwd :- (schema/maybe [schema/Str])
```

Comment:
I think this should just be `[schema/Str]` - I'm pretty sure the `maybe` does nothing here?

```clojure
=> (schema/validate [schema/Str] nil)
nil
```